hack/app_sre_pr_check.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Is this file going to be used at all? I understand this operator will run PR checks via Prow

Reply:
as of now the operator in app-sre is pointing to this script.

if it's in prow that's great! It's better to be tested twice then none at all.

also IIUC this is the entrypoint if the make target doesn't exist, so the file is required

Reply:
oh, it's PR check.. did'nt read it good the first time.. will delete